### DISCUSSION TITLE
Prevent execution of a cell

### DISCUSSION THREAD

**Discussion by whitead** [executablebooks/meta#122] [Q&A]
I would like to have a cell appear to help users install packages when launching the ipynb on binder or Google Colab. I do not want the cell to be executed when building because it contains code to install miniconda that is necessary on Google Colab. How do I achieve this? Thank you!!

**Comment by chrisjsewell** [comment]
I think what you after is https://github.com/executablebooks/jupyter-book/issues/833,
which is not addressed yet 😢
But obviously the more upvotes it gets, the more likely it will be 😉
